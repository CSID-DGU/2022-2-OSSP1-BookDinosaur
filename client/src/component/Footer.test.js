import render from '@testing-library/react';
import Footer from './Footer';

test('matches snapshot', () => {
    const utils = render(
        <Footer />
    )
    expect(utils.container).toMatchSnapshot();
})
