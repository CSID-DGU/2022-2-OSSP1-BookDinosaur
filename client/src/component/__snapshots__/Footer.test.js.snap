// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div>
    <h4>
      Copyright 2022. Read Lead. all rights reserved.
    </h4>
    <br />
  </div>
</div>
`;
